underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -149,29 +152,56 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri) {
     }
 
     AmazonS3Client amazonS3Client = new AmazonS3Client(credentials, clientConf);
+    // Set your endpoint.
```

Comment:
NIT: instead of saying "your" you can say "Set the custom endpoint".

Also, could you put that within the `if` statement?